sdk/api-learn/Azure.Learn.AppConfig/tests/LearnAppConfigEnvironment.cs

```diff
@@ -1,14 +1,14 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
 using Azure.Core.TestFramework;
 
-namespace Azure.Learn.AppConfig.Samples
+namespace Azure.Learn.AppConfig.Tests
 {
     // To learn more about TestEnvironment classes, please see: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core.TestFramework/README.md
-    public class AppConfigClientTestEnvironment : TestEnvironment
+    public class LearnAppConfigEnvironment : TestEnvironment
```

Comment:
nit:  Non-impactful, but the convention that I've seen is to use the form ``ServiceNameTestEnvironment`.